agent/engine/docker_task_engine_test.go

```diff
@@ -156,6 +156,7 @@ func TestBatchContainerHappyPath(t *testing.T) {
 	default:
 	}
 	eventsReported.Wait()
+	client.EXPECT().DescribeContainer(gomock.Any()).AnyTimes()
```

Comment:
Sort of.  `CheckTaskState` gets called as part of a "steady state verification" that (in normal cases) occurs every 10 minutes.  In this test code, the verification is triggered by a write to the `steadyStateVerify` channel, which occurs on line 167.  Because the test is specifically triggering this code, it appears to be that the maximum number of times `DescribeContainer` is invoked is once.

`AnyTimes` is an anti-pattern in cases like this (that applies to the existing code as well, not just your change); we'd like to ensure that the task engine is functioning as expected.